web/cypress/integration/user/login.spec.js

```diff
@@ -44,4 +44,11 @@ context('Login Test', () => {
     cy.contains('Login').click();
     cy.get('.ant-notification-notice-message').should('contain', 'Successfully');
   });
+
+  it('Press Enter to login success', () => {
+    cy.visit('/user/Login');
+    cy.get('#control-ref_username').type('user');
+    cy.get('#control-ref_password').type('user{enter}');
```

Comment:
`user{enter}`?

Reply:
Yes, there are examples in the official documentation.
![image](https://user-images.githubusercontent.com/72343596/107878725-8286d880-6f0f-11eb-85d6-ed485fd2bfa0.png)

Reply:
what's `enter`?

Reply:
![image](https://user-images.githubusercontent.com/72343596/107880395-64be7100-6f19-11eb-8b1c-be478b521d53.png)
Information can be found here https://docs.cypress.io/api/commands/type.html#Events